bin/epaa.py

```diff
@@ -638,12 +672,13 @@ def get_protein_ids_for_transcripts(idtype, transcripts, ensembl_url, reference)
 
     # too long requests will fail
     if len(transcripts) > 400:
```

Comment:
```suggestion
    if len(transcripts) > 200:
```

Reply:
We change this number to make the predictions more stable for large number of variants. In these cases it (randomly) fails due to `HTTP 500` coming from `Ensembl` BioMart. One idea was to use a `Ensembl` mirror which does not work in this case since "archive" versions are not available there. We will come up with a more stable and sustainable solution in the next release.